### PR TITLE
HBASE-22896 TestHRegion.testFlushMarkersWALFail is flaky

### PR DESCRIPTION
**Pull request by sunhelly**

### CODE REVIEW

File: hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java [hunk 1]
```diff
@@ -1233,10 +1232,10 @@ public long getLength() {
     this.region = initHRegion(tableName, HConstants.EMPTY_START_ROW,
       HConstants.EMPTY_END_ROW, false, Durability.USE_DEFAULT, wal, family);
     region.put(put);
-
+
```
Comment (Apache-HBase) [inline hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java]: whitespace:end of line

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 159 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 432 | master passed |
| +1 | compile | 73 | master passed |
| +1 | checkstyle | 105 | master passed |
| +1 | shadedjars | 357 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 39 | master passed |
| 0 | spotbugs | 276 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 275 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 336 | the patch passed |
| +1 | compile | 59 | the patch passed |
| +1 | javac | 59 | the patch passed |
| +1 | checkstyle | 92 | the patch passed |
| -1 | whitespace | 0 | The patch has 1 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply |
| +1 | shadedjars | 326 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1119 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 36 | the patch passed |
| +1 | findbugs | 277 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 16557 | hbase-server in the patch failed. |
| +1 | asflicense | 25 | The patch does not generate ASF License warnings. |
| | | 20688 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestSnapshotCloneIndependence |
|   | hadoop.hbase.client.TestAsyncRegionAdminApi2 |
|   | hadoop.hbase.replication.TestReplicationKillSlaveRS |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux e9951fefa59a 4.15.0-48-generic #51-Ubuntu SMP Wed Apr 3 08:28:49 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / 56980fb097 |
| Default Java | 1.8.0_181 |
| whitespace | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/2/artifact/out/whitespace-eol.txt |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/2/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/2/testReport/ |
| Max. process+thread count | 4636 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 35 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 411 | master passed |
| +1 | compile | 63 | master passed |
| +1 | checkstyle | 86 | master passed |
| +1 | shadedjars | 301 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 37 | master passed |
| 0 | spotbugs | 271 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 269 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 352 | the patch passed |
| +1 | compile | 63 | the patch passed |
| +1 | javac | 63 | the patch passed |
| +1 | checkstyle | 91 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 338 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1044 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 35 | the patch passed |
| +1 | findbugs | 255 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 9496 | hbase-server in the patch passed. |
| +1 | asflicense | 27 | The patch does not generate ASF License warnings. |
| | | 13271 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux f35ea551b6a5 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / 56980fb097 |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/3/testReport/ |
| Max. process+thread count | 4820 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/3/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 117 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 407 | master passed |
| +1 | compile | 65 | master passed |
| +1 | checkstyle | 92 | master passed |
| +1 | shadedjars | 304 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 41 | master passed |
| 0 | spotbugs | 279 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 277 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 342 | the patch passed |
| +1 | compile | 60 | the patch passed |
| +1 | javac | 60 | the patch passed |
| +1 | checkstyle | 92 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 302 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1059 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 37 | the patch passed |
| +1 | findbugs | 281 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 14128 | hbase-server in the patch failed. |
| +1 | asflicense | 25 | The patch does not generate ASF License warnings. |
| | | 18051 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.7 Server=18.09.7 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 9eccbfdff1e6 4.15.0-52-generic #56-Ubuntu SMP Tue Jun 4 22:49:08 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / 83e77940c5 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/4/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/4/testReport/ |
| Max. process+thread count | 4706 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/4/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by infraio** [comment]
It still is flaky after I revert HBASE-22857?

**Comment by sunhelly** [comment]
Thanks @infraio .

> It still is flaky after I revert HBASE-22857?

Yes, it throws NPE when we run testFlushMarkersWALFail.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 45 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 435 | master passed |
| +1 | compile | 68 | master passed |
| +1 | checkstyle | 79 | master passed |
| +1 | shadedjars | 278 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 37 | master passed |
| 0 | spotbugs | 245 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 244 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 299 | the patch passed |
| +1 | compile | 56 | the patch passed |
| +1 | javac | 56 | the patch passed |
| +1 | checkstyle | 75 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 286 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 953 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 36 | the patch passed |
| +1 | findbugs | 255 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 10258 | hbase-server in the patch passed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 13812 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux d9a026e531c7 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / 090c55f3ff |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/5/testReport/ |
| Max. process+thread count | 4453 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/5/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 50 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 404 | master passed |
| +1 | compile | 64 | master passed |
| +1 | checkstyle | 92 | master passed |
| +1 | shadedjars | 337 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 42 | master passed |
| 0 | spotbugs | 270 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 267 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 375 | the patch passed |
| +1 | compile | 64 | the patch passed |
| +1 | javac | 64 | the patch passed |
| +1 | checkstyle | 90 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 337 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1172 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 42 | the patch passed |
| +1 | findbugs | 284 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 9412 | hbase-server in the patch passed. |
| +1 | asflicense | 34 | The patch does not generate ASF License warnings. |
| | | 13528 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/6/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 5f3b8bacf823 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / 090c55f3ff |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/6/testReport/ |
| Max. process+thread count | 4887 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/6/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 40 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 494 | master passed |
| +1 | compile | 61 | master passed |
| +1 | checkstyle | 98 | master passed |
| +1 | shadedjars | 306 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 37 | master passed |
| 0 | spotbugs | 267 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 265 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 331 | the patch passed |
| +1 | compile | 58 | the patch passed |
| +1 | javac | 58 | the patch passed |
| +1 | checkstyle | 88 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 310 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1049 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 37 | the patch passed |
| +1 | findbugs | 279 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 10236 | hbase-server in the patch passed. |
| +1 | asflicense | 38 | The patch does not generate ASF License warnings. |
| | | 14141 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/7/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 992df9b9a952 4.15.0-54-generic #58-Ubuntu SMP Mon Jun 24 10:55:24 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / ee5cf0d209 |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/7/testReport/ |
| Max. process+thread count | 4521 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/7/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 37 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 325 | master passed |
| +1 | compile | 55 | master passed |
| +1 | checkstyle | 79 | master passed |
| +1 | shadedjars | 274 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 38 | master passed |
| 0 | spotbugs | 237 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 235 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 301 | the patch passed |
| +1 | compile | 53 | the patch passed |
| +1 | javac | 53 | the patch passed |
| +1 | checkstyle | 78 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 272 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 910 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 36 | the patch passed |
| +1 | findbugs | 240 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 9238 | hbase-server in the patch failed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 12574 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.replication.TestReplicationDroppedTables |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/8/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux d5f719c506b0 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / b642ee0418 |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/8/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/8/testReport/ |
| Max. process+thread count | 5251 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/8/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 32 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 361 | master passed |
| +1 | compile | 63 | master passed |
| +1 | checkstyle | 90 | master passed |
| +1 | shadedjars | 304 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 41 | master passed |
| 0 | spotbugs | 272 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 269 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 333 | the patch passed |
| +1 | compile | 60 | the patch passed |
| +1 | javac | 60 | the patch passed |
| +1 | checkstyle | 88 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 306 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1058 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 37 | the patch passed |
| +1 | findbugs | 288 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 10394 | hbase-server in the patch failed. |
| +1 | asflicense | 35 | The patch does not generate ASF License warnings. |
| | | 14168 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.regionserver.TestHRegionWithInMemoryFlush |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/9/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux a79b55e5825d 4.15.0-54-generic #58-Ubuntu SMP Mon Jun 24 10:55:24 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / 97fbaa658e |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/9/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/9/testReport/ |
| Max. process+thread count | 4478 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/9/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 32 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 356 | master passed |
| +1 | compile | 61 | master passed |
| +1 | checkstyle | 88 | master passed |
| +1 | shadedjars | 303 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 37 | master passed |
| 0 | spotbugs | 239 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 237 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 330 | the patch passed |
| +1 | compile | 58 | the patch passed |
| +1 | javac | 58 | the patch passed |
| +1 | checkstyle | 87 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 305 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1057 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 36 | the patch passed |
| +1 | findbugs | 281 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 10914 | hbase-server in the patch failed. |
| +1 | asflicense | 26 | The patch does not generate ASF License warnings. |
| | | 14614 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.regionserver.TestHRegionWithInMemoryFlush |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/10/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 6b348cc6d259 4.15.0-54-generic #58-Ubuntu SMP Mon Jun 24 10:55:24 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / a5ef6b287c |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/10/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/10/testReport/ |
| Max. process+thread count | 4489 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/10/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 42 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 336 | master passed |
| +1 | compile | 56 | master passed |
| +1 | checkstyle | 81 | master passed |
| +1 | shadedjars | 279 | branch has no errors when building our shaded downstream artifacts. |
| +1 | javadoc | 37 | master passed |
| 0 | spotbugs | 259 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 256 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 302 | the patch passed |
| +1 | compile | 55 | the patch passed |
| +1 | javac | 55 | the patch passed |
| +1 | checkstyle | 79 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 270 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 941 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2. |
| +1 | javadoc | 37 | the patch passed |
| +1 | findbugs | 272 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 10552 | hbase-server in the patch failed. |
| +1 | asflicense | 33 | The patch does not generate ASF License warnings. |
| | | 14001 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.regionserver.TestHRegionWithInMemoryFlush |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/11/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/551 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux d2fba61ee65f 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-551/out/precommit/personality/provided.sh |
| git revision | master / a5ef6b287c |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/11/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/11/testReport/ |
| Max. process+thread count | 4512 (vs. ulimit of 10000) |
| modules | C: hbase-server U: hbase-server |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-551/11/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**